### PR TITLE
feat: curioweb: Show piece info on the sector page

### PR DESCRIPTION
**Pull request by magik6k**
## Related Issues
Needed to debug some issues on my cluster

## Additional Info
obraz

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: lib/harmony/harmonydb/sql/20240501-harmony-indexes.sql
Comment (magik6k) [inline lib/harmony/harmonydb/sql/20240501-harmony-indexes.sql:1]: This is kinda a drive-by fix, adding this reduced load on my DB quite significantly (optimized the heaviest query in harmonytask into one of the lighter ones)

**Review by LexLuthr — APPROVED** [review]
